src/main/java/io/jenkins/plugins/coverage/adapter/JavaCoverageReportAdapterDescriptor.java

```diff
@@ -0,0 +1,22 @@
+package io.jenkins.plugins.coverage.adapter;
+
+import io.jenkins.plugins.coverage.targets.CoverageElement;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class JavaCoverageReportAdapterDescriptor extends CoverageReportAdapterDescriptor<CoverageReportAdapter> {
+
+    public JavaCoverageReportAdapterDescriptor(Class<? extends CoverageReportAdapter> clazz) {
+        super(clazz);
+    }
+
+    @Override
+    public List<CoverageElement> getCoverageElements() {
+        return Arrays.asList(new CoverageElement("Java Group", 0),
+                new CoverageElement("Java Package", 1),
```

Comment:
It may be a proble. `Cobertura` plugin and other such plugin are widely used to report coverage of non-Java code, even with code browsing.  I would advice to have a way to specify the code type in the implementations